apps/dashboard/app/controllers/active_jobs_controller.rb

```diff
@@ -75,11 +75,13 @@ def get_job(jobid, cluster)
       data = cluster.job_adapter.info(jobid)
 
       raise OodCore::JobAdapterError if data.native.nil?
-      Jobstatusdata.new(data, cluster, true)
+      ActiveJobs::Jobstatusdata.new(data, cluster, true)
 
     rescue OodCore::JobAdapterError
       OpenStruct.new(name: jobid, error: "No job details because job has already left the queue." , status: status_label("completed") )
     rescue => e
+      Rails.logger.info("#{e}:#{e.message}")
+      Rails.logger.info(e.backtrace.join("\n"))
```

Comment:
I added this during testing, but I could live with keeping it. Otherwise it silently fails.

Reply:
(though `info` is probably the wrong level to use)